.github/workflows/ci.yml

```diff
@@ -191,8 +195,12 @@ jobs:
         with:
           java-version: 8
           distribution: "temurin"
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.8
       - name: Run CI
-        run: ./ci/run_ci.sh integration_tests
+        run: python ./ci/run_ci.py java --version integration_tests
```

Comment:
It looks a little odd to let integration_tests as a version

Reply:
We could rename it `version_or_test` ?

Reply:
how about add a --module  or --component to specify which module to run in java, if no such module specified, we run all modules for java